drivers/dac/ad5791/ad5791.h

```diff
@@ -61,6 +61,16 @@ enum ad5791_type {
 	ID_AD5791
 };
 
+/* Possible voltage spans for linearity error compensation */
+enum ad5791_lin_comp_select {
+	AD5781_SPAN_UPTO_10V,
+	AD5791_SPAN_10V_TO_12V,
+	AD5791_SPAN_12V_TO_16V,
+	AD5791_SPAN_16V_TO_19V,
+	AD5791_SPAN_19V_TO_20V,
+	AD5781_SPAN_10V_TO_20V = 4
```

Comment:
if you provide value for one enum member, then all members should be set explicitly.